portfolio_prices.py

```diff
@@ -0,0 +1,65 @@
+import investpy as inv
+from datetime import datetime
+import pandas as pd
+import numpy as np
+
+today = datetime.today().strftime("%d/%m/%Y")
+
+
+def get_assets_data_frames(assets, asset_function, country, start_date, end_date):
+

```

Comment:
I don't think is necessary so many spaces in here, we don't have a checkstyle or something to keep track of this problem. I don't think spaces are necessary. But if you want to use it, I suggest the following:

- declaring variables
<space>
- operations
<space>
- return

Reply:
I don't like these spaces too, but autopep8 that insert it.